packages/docmanager-extension/src/index.ts

```diff
@@ -784,7 +784,7 @@ function addLabCommands(
 
   commands.addCommand(CommandIDs.rename, {
     label: () =>
-      trans.__('Rename %1…', fileType(contextMenuWidget(), docManager)),
+      trans.__('Rename%1…', fileType(contextMenuWidget(), docManager)),
```

Comment:
Sorry, we still need that space before the `%1` too :).

Reply:
No problem I'll put it back in now.

Reply:
Ah, now I understand why you took it out. If the file type is empty, you end up with that extra space. How about leaving it as `Rename%1`, and put the space in the replacement text? Like this:
```suggestion
      trans.__('Rename%1…', ' ' + fileType(contextMenuWidget(), docManager)),
```

Reply:
(You can see the effect by opening a notebook, say, and then looking at the menu item. Then it should say something like "Rename Notebook..."

Reply:
I see what you mean. I believe I updated this to match your suggested change :)